### PR TITLE
src/main: display Progress and LastError D-Bus properties in client

### PR DESCRIPTION
**Pull request by ejoerns**
This patch makes progress and LastError information available from the
command line client that only printed

    rauc-Message: installing /bundle.raucb: installing
    installing `/bundle.raucb` failed

This will now make it be much more verbose, such as

    rauc-Message: installing /bundle.raucb: 000% Installing
    rauc-Message: installing /bundle.raucb: 000% Determining slot states
    rauc-Message: installing /bundle.raucb: 020% Determining slot states done.
    rauc-Message: installing /bundle.raucb: 020% Checking bundle
    [...]
    rauc-Message: installing /bundle.raucb: 050% Loading manifest file done.
    rauc-Message: installing /bundle.raucb: 060% Verifying manifest failed.
    rauc-Message: installing /bundle.raucb: 100% Installing failed.
    rauc-Message: installing /bundle.raucb: LastError: Failed verifying manifest: Invalid compatible: 'A' (mf) does not match 'AA' (sys)
    installing `/bundle.raucb` failed

Signed-off-by: Enrico Joerns <ejo@pengutronix.de>

### CODE REVIEW

**Comment by ejoerns** [comment]
This fixes #108

File: src/main.c [hunk 1]
```diff
@@ -65,6 +67,10 @@ static void on_installer_changed(GDBusProxy *proxy, GVariant *changed,
 	g_mutex_lock(&args->status_mutex);
 	if (g_variant_lookup(changed, "Operation", "s", &msg)) {
 		g_queue_push_tail(&args->status_messages, g_strdup(msg));
+	} else if (g_variant_lookup(changed, "Progress", "(isi)", &percentage, &message, &depth)) {
+		g_queue_push_tail(&args->status_messages, g_strdup_printf("%03"G_GINT32_FORMAT"%% %s", percentage, message));
```
Comment (jluebbe) [inline src/main.c]: Please pad using spaces instead of zeros.

Comment (ejoerns) [inline src/main.c]:
``` diff
diff --git a/src/main.c b/src/main.c
index 284e574..3093a67 100644
--- a/src/main.c
+++ b/src/main.c
@@ -68,7 +68,7 @@ static void on_installer_changed(GDBusProxy *proxy, GVariant *changed,
        if (g_variant_lookup(changed, "Operation", "s", &msg)) {
                g_queue_push_tail(&args->status_messages, g_strdup(msg));
        } else if (g_variant_lookup(changed, "Progress", "(isi)", &percentage, &message, &depth)) {
-               g_queue_push_tail(&args->status_messages, g_strdup_printf("%03"G_GINT32_FORMAT"%% %s", percentage, message));
+               g_queue_push_tail(&args->status_messages, g_strdup_printf("%3"G_GINT32_FORMAT"%% %s", percentage, message));
        } else if (g_variant_lookup(changed, "LastError", "s", &message) && message[0] != '\0') {
                g_queue_push_tail(&args->status_messages, g_strdup_printf("LastError: %s", message));
        }

```

**Comment by ejoerns** [comment]
rebased onto current master and fixed zero-padding of percentages

**Pull request merged**